tokio/src/io/poll_evented.rs

```diff
@@ -83,7 +91,7 @@ impl<E: Source> PollEvented<E> {
     #[track_caller]
     #[cfg_attr(feature = "signal", allow(unused))]
     pub(crate) fn new(io: E) -> io::Result<Self> {
-        PollEvented::new_with_interest(io, Interest::READABLE | Interest::WRITABLE)
+        PollEvented::new_with_interest(io, ALL_INTEREST)
```

Comment:
I would not change this, as this changes existing behaviour and I think `Interest::READABLE | Interest::WRITABLE` is a sane default most users would want.

Reply:
makes sense. I added a line to the docs specifying that only read and write interests are configured